payments-core/src/main/java/com/stripe/android/cards/DefaultStaticCardAccountRanges.kt

```diff
@@ -102,21 +102,27 @@ class DefaultStaticCardAccountRanges : StaticCardAccountRanges {
             )
         }
 
+        // https://www.discoverglobalnetwork.com/content/dam/discover/en_us/dgn/docs/IPP-VAR-Enabler-Compliance.pdf
```

Comment:
Is there a better source for getting UnionPay bin numbers?

Reply:
I wonder if it's worth downloading this PDF and dropping it somewhere internal? I feel like it could get lost pretty easily.